README.md

```diff
@@ -43,562 +49,131 @@ There are two interactive tutorials provided,
-
-The two examples go into some more detail regarding this method.
-
-Bugs or Limitations
--------------------
-
-- Sometimes line group and HSL highlighting may appear out of sync if you've
-  applied undo/redo chains to a lush.ify'd file. Generally typing more into the
-  buffer will fix these issues as the highlighter re-syncs with the buffer
-  state.
-
-- You may find some elements don't update in real time (LSP sign column for
-  example). This is a side effect of colours are applied to those elements,
-  only as they are created (I believe).
-  The group name in your lush-spec should update to let you see how it will
-  look when your theme is loaded.
-
-- Lush.ify'd `hsl()` and group name highlight may sometimes be obscured by
-  CursorLine highlighting. If this is a problem, you can set CursorLine to an
-  empty definition or disable the cursor line with `set nocursorline`.
-
-Todo / Future ideas
--------------------
-
-**Theme Inheritance**
-
-This would give you the ability to require someone else's theme as a base to
-yours.
-
-Imagine you really like a theme, but wish the background was a bit different,
-you could define your lush-spec with that theme as base/parent and just change
-the groups you want.
-
-I have a draft of this written but was unsure how useful it would really be as
-a feature and if it was worth the work hours. Perhaps if Lush were popular this
-more sense.
-
-**Global HSL Shifting / Contrast Shifting?**
-
-Unsure how useful this would be in the real world, but switching between some
-machines can render some colourschemes differently, because their screens
-or terminals are different.
-
-The idea would be you could set a global shift on HSL to effect all colours
-that are pushed through it.
-
-In actuality, I think what I *really* want is a contrast scale, which isn't as
-simple as simply "make it all brighter" or "make it all bluer".
-
-**Automatic Property Inference**
-
-Would allow for syntax like:
-
-```lua
--- automatically infer appropriate key (Normal.fg)
-CursorLine { fg = Normal, bg = Visual }
-```
-
-Most of this code is actually already present, but the ability to write
-`fg = Normal` tends to encourage `fg = Normal.ro(...)` at a later time,
-is an invalid operation.
-
-Without a uniform solution to this, I'm reticent to "muddy" the API.
-
-For now, you must write `fg = Normal.fg`.
-
-Change Log
-----------
+You can then yank the contents of the buffer, and paste it in `cool_name/colors/cool_name.vim`.
+Remember to change `lush_theme.cool_name` to the name of the file where your lush-spec is defined.
 
-- 2020-11-23
-  - Lush.ify now reports errors in a more consistent format.
-  - Lush.ify now rate-limits eval attempts on parsing errors.
-- 2020-11-21
-  - Lush-spec now supports group inheritance.
-- 2020-11-19
-  - Initial release.
+Lush also provides functions to export your color scheme in a more automated way.
+See the related documentation. <!-- link to parse-compile-apply -->
```

Comment:
Yep, probably link out to the vim docs or API.md or something. I will handle that once the readme settles.